.travis.yml

```diff
@@ -1,18 +1,25 @@
 language: scala
-sudo: false
+branches:
+  except:
+  - release
+sudo: true
 scala:
-  - 2.11.12
-  - 2.12.6
+- 2.11.12
+- 2.12.6
```

Comment:
Should this be changed to `2.12.7`? `build.sbt` currently points to `2.12.7`

Reply:
Oh yeah, we totally should.

Reply:
One pro for tossing away a build matrix is we never have to do that again.